src/bin/tattoo.ts

```diff
@@ -13,13 +13,14 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-import {CliOptions, getCommandLineOptions, showCliHelp, loadConfigFileOptions, mergeConfigFileOptions, ensureGitHubToken} from '../cli';
+import {CliOptions, getCommandLineOptions, showCliHelp, showVersion, loadConfigFileOptions, mergeConfigFileOptions, ensureGitHubToken} from '../cli';
 import {Runner, RunnerOptions} from '../runner';
 
 async function main() {
   console.time('tattoo');
   try {
     const cliOptions: CliOptions = getCommandLineOptions();
+    showVersion(cliOptions);
```

Comment:
I always have trouble following these because it's unclear that they have the ability to exit the process. Can we pull the `options.version` check and the `exit` call out of `showVersion()`, and have it only be responsible for printing the version? Ditto for `showCliHelp()`